Documentation/guides/AndroidAssetPacks.md

```diff
@@ -0,0 +1,176 @@
+resources. This type of `feature` pack can be installed at install-time,
+fast-follow or ondemand. It is most useful for apps which contain allot
+of `Assets`, such as Games or Multi Media applications.
+See the [documentation](https://developer.android.com/guide/playcore/asset-delivery) for details on how this all works.
+.NET Android does not have any official support for this type of pack.
+However a hack is available via the excellent @infinitespace-studios on
+[github](https://github.com/infinitespace-studios/MauiAndroidAssetPackExample).
+This hack allows developers to place additional assets in a special
+`NoTargets` project. This project is built just after the final `aab` is
+produced. It builds a zip file which is then added to the `@(Modules)`
+ItemGroup in the main application. This zip is then included into the
+final app as an additional feature.
+
+## Asset Pack Specification
+
+We want to provide our users the ability to use `Asset` packs without
+having to implement the hack provided by @infinitespace-studios. Using
+a separate project like in the hack is one way to go. It does have some
+issues though.
+
+1. It is a `special` type of project. It requires a `global.json` which imports the
+   `NoTargets` sdk.
+2. There is no IDE support for building this type of project.
+
+Having the user go through a number of hoops to implement this for
+.NET Android or .net Maui is not ideal. We need a simpler method.
+
+The new idea is to make use of additional metadata on `AndroidAsset`
+Items to allow the build system to split up the assets into packs
+automatically. So it is proposed that we implement support for something
+like this
+
+```xml
+<ItemGroup>
+   <AndroidAsset Include="Asset/data.xml" />
+   <AndroidAsset Include="Asset/movie.mp4" AssetPack="assets1" />
+   <AndroidAsset Include="Asset/movie2.mp4" AssetPack="assets1" />
+</ItemGroup>
+```
+
+In this case the additional `AssetPack` attribute is used to tell the
+build system which pack to place this asset in. Since auto import of items
+is common now we need a way for a user to add this additional attribute
+to auto included items. Fortunately we are able to use the following.
+
+```xml
+<ItemGroup>
+   <AndroidAsset Update="Asset/movie.mp4" AssetPack="assets1" />
+   <AndroidAsset Update="Asset/movie2.mp4" AssetPack="assets1" />
+   <AndroidAsset Update="Asset/movie3.mp4" AssetPack="assets2" />
+</ItemGroup>
+```
+
+This code uses the `Update` attribute to tell MSBuild that we are going
+to update a specific item. Note in the sample we do NOT need to include
+an `Update` for the `data.xml`, since this is auto imported it will still
+end up in the main feature in the aab.
+
+Additional attributes can be used to control what type of asset pack is
+produced. The only extra one supported at this time is `DeliveryType`,
+this can have a value of `InstallTime`, `FastFollow` or `OnDemand`.
+Additional attributes do not need to be included on ALL items. Any one
```

Comment:
eh?  So if I set `%(AndroidAsseet.DeliveryType)` on one item, all other `@(AndroidAsset)` values get the same metadata value?  Within what scope: all other `@(AndroidAsset)` values in the same file?  In the entire enclosing `.csproj`?  In the final app?

I'm not at all sure what this is supposed to mean, or the ramifications, other than mild horror.